polymorphic/tests.py

```diff
@@ -15,9 +19,14 @@
 from django.contrib.contenttypes.models import ContentType
 from django.utils import six
 
-from polymorphic import PolymorphicModel, PolymorphicManager, PolymorphicQuerySet
+from polymorphic.models import PolymorphicModel
+from polymorphic.manager import PolymorphicManager
+from polymorphic.query import PolymorphicQuerySet
 from polymorphic import ShowFieldContent, ShowFieldType, ShowFieldTypeAndContent
-from polymorphic.tools_for_tests import UUIDField
+if django.VERSION < (1,8):
+    from polymorphic.tools_for_tests import UUIDField
+else:
+    from django.db.models import UUIDField
```

Comment:
Perhaps handle this with an try..except too, like the other code? (and comment that it's for django 1.9+)